src/test/java/com/wikia/webdriver/common/dataprovider/ads/GermanAdsDataProvider.java

```diff
@@ -41,59 +41,7 @@ private GermanAdsDataProvider() {
   }
 
   @DataProvider
-  public static final Object[][] pagesForIVW2Corporate() {
```

Comment:
Let's leave the input data in data provider and merge the three sections to one.